### ISSUE TITLE
Phone field prefix improve interaction and a11y

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#1621]
## Description
Improve documentation of phone field prefix

the phone field has web interaction problems because you can input date if you click on prefix area.

Include documentation in specs the way to solve this interaction

Should be possible to input data in prefix?

## Tasks

- [ ] Update specs
- [ ] Web ticket
